code/modules/reagents/reagent_containers/syringes.dm

```diff
@@ -181,7 +181,7 @@
 
 
 /obj/item/weapon/reagent_containers/syringe/update_icon()
-	var/rounded_vol = min(max(round(reagents.total_volume,5),5),15)
+	var/rounded_vol = round(reagents.total_volume,5)

```

Comment:
The check you removed was making sure that `rounded_vol` was always between 5 and 15, so that meant we never got the 0 value, which is what we wanted.

What you did here was you removed all checks, so now we can get any multiple of 5 (including negative numbers, and numbers over 15, both of which have no icons).

What I suggested was replacing the line with `var/rounded_vol = Clamp(round(reagents.total_volume,5), 0, 15)`, which is the same thing as the original line, but instead of clamping the `rounded_vol` between 5 and 15, it clamps it between 0 and 15, which is what we want to fix the issue.